packages/headless/src/api/platform-client.ts

```diff
@@ -44,8 +44,15 @@ export const NoopPreprocessRequestMiddleware: PreprocessRequestMiddleware = (
   request
 ) => request;
 
+export type PlatformClientCallError =
+  | ExpiredTokenError
+  | DisconnectedError
+  | Error;
+
 export class PlatformClient {
-  static async call(options: PlatformClientCallOptions): Promise<Response> {
+  static async call(
+    options: PlatformClientCallOptions
+  ): Promise<Response | PlatformClientCallError> {
```

Comment:
Although I "changed" the return type here, an `Error` object was already occasionally being returned by `call`; the reason TypeScript didn't complain was because it had the `any` type.